Country.py

```diff
@@ -44,15 +58,15 @@ def infected_ratio(self):
 
     def death(self):
         """
-        A part of the infected population would be passed away.
+        A part of the infected population would be killed.
         Then the infected population and maximum population will be reduced as many as the number of people death.
         """
         death_pop = 0
         alive_pop = self.max_pop
-        if self.infected_ratio() > 0.10:
-            if self.infected_pop > 10:
+        if self.infected_ratio() > 0.10:                   #ratios are arbitrarily selected
```

Comment:
It's recommended that you give constants a name and capitalize them. `self.infected_ratio() > THRESHOLD`. 
Also, it's common practice to make a config class that contains all these arbitrary values